From de616f65996dc559f4153cba679479ff8ff5d551 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 24 Aug 2020 14:39:36 +0800
Subject: [PATCH] Test case5 for Change guid value in Ppis

Description: Change guid value in Ppis section (to gEfiVectorHandoffInfoPpiGuid = { 0x3cd638e4, 0x6d33, 0x4dce, { 0x89, 0xdb, 0x83, 0x4f, 0x97, 0xdd, 0xf3, 0xce }}) in MdeModulePkg/MdeModulePkg.dec.

Repo: edk2

Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdePkg/MdePkg.dec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MdePkg/MdePkg.dec b/MdePkg/MdePkg.dec
index 682e61cb88..70be9676c2 100644
--- a/MdePkg/MdePkg.dec
+++ b/MdePkg/MdePkg.dec
@@ -911,7 +911,7 @@
   gEfiTemporaryRamDonePpiGuid        = { 0xceab683c, 0xec56, 0x4a2d, { 0xa9, 0x6, 0x40, 0x53, 0xfa, 0x4e, 0x9c, 0x16 } }
 
   ## Include/Ppi/VectorHandoffInfo.h
-  gEfiVectorHandoffInfoPpiGuid       = { 0x3cd652b4, 0x6d33, 0x4dce, { 0x89, 0xdb, 0x83, 0xdf, 0x97, 0x66, 0xfc, 0xca }}
+  gEfiVectorHandoffInfoPpiGuid       = { 0x3cd638e4, 0x6d33, 0x4dce, { 0x89, 0xdb, 0x83, 0x4f, 0x97, 0xdd, 0xf3, 0xce }}
 
   ## Include/Ppi/IsaHc.h
   gEfiIsaHcPpiGuid                   = { 0x8d48bd70, 0xc8a3, 0x4c06, {0x90, 0x1b, 0x74, 0x79, 0x46, 0xaa, 0xc3, 0x58 } }
-- 
2.27.0.windows.1

